--- plugins/module_utils/base_module.py.orig	2026-09-01 13:32:56.027043144 +0200
+++ plugins/module_utils/base_module.py	2026-09-01 13:32:09.731251103 +0200
@@ -69,28 +69,7 @@
         return f'{base_url}/proxy/{self.get_name()}/integration'
 
     def get_spec_url(self, version: str) -> str:
-        application = self.get_name()
-        spec_url = f"{OPENAPI_SPEC_BASE_URL}/{application}/v{version}/integration.json"
-
-        if not os.path.exists(spec_url):
-            self.module.warn(f"Spec for version {version} not found, falling back to latest available version.")
-            versions_endpoint = f"{DEV_PORTAL_BASE_URL}/v1/api-docs?application={application}"
-            resp = self.rt.call(versions_endpoint)
-            versions_data = resp.json().get("data", []).get("versions", [])
-            available_versions = [v.get("version").lstrip("v") for v in versions_data if "version" in v]
-
-            # Find the closest version (<= requested version)
-            closest_version = None
-            for v in sorted(available_versions, reverse=True):
-                if v <= version:
-                    closest_version = v
-                    break
-            if not closest_version and available_versions:
-                closest_version = available_versions[-1]  # fallback to oldest
-
-            version = closest_version
-
-        return f"{OPENAPI_SPEC_BASE_URL}/{application}/v{version}/integration.json"
+        return f"https://raw.githubusercontent.com/beezly/unifi-apis/refs/heads/main/unifi-network/{version}.json"
 
     def load_or_update_spec(self, version: str):
         cache_dir = os.path.join(tempfile.gettempdir(), "ubiquiti.unifi_api")
